src/test/java/org/opensearch/sdk/FailingTransport.java

```diff
@@ -0,0 +1,112 @@
+package org.opensearch.sdk;
+
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
Was this sourced from other code that had an Elasticsearch license header?  Probably want to add a short comment explaining why its header is different than everything else in this package, and probably want to put such code in its own package so it can be excluded from the spotless header rewrite.

Reply:
+1

Reply:
Yes, this is from the old ES code and that's why the header.

Reply:
Can we avoid copying in like this an instead pull references from the OpenSearch core project?

Reply:
This is from the [java-client](https://github.com/opensearch-project/opensearch-java/blob/main/java-client/src/test/java/org/opensearch/client/documentation/FailingTransport.java) and is not published to maven to consume.

Reply:
> This is from the [java-client](https://github.com/opensearch-project/opensearch-java/blob/main/java-client/src/test/java/org/opensearch/client/documentation/FailingTransport.java) and is not published to maven to consume. Can we remove the ES header ...

The link you gave to the java-client includes an ES header, so the answer requires further research.

If the code you copied was part of the original fork from ES then no, you cannot remove the ES header.  ES maintains the copyright and licenses it to you.

If the code you copied was added under AL2.0 after the fork, then it is under the "modifications" clause of that copyright header and you could remove it.

Reviewing the history of this contribution it looks like it originated from this commit under ES license:
https://github.com/opensearch-project/opensearch-java/commit/4f3a394136652d5191eddb22aa6a944455fd7128

So you must retain the ES header.